cpp/ql/src/change-notes/2025-03-11-basic-int-types.md

```diff
@@ -0,0 +1,4 @@
+---
+category: majorAnalysis
```

Comment:
```suggestion
category: minorAnalysis
```
I think minor is fine here, as these queries usually aren't run.

Reply:
Agreed!